HOWTO-RELEASE

```diff
@@ -32,47 +45,191 @@
+to master.
+
+
+1.3 Write release notes
+-------------------------------------------------------------------------------
+
+Update `NEWS` with descriptions of the changes since the last release.
+Hopefully issues and pull requests that go into the new release have been
+properly tagged with the milestone for the current release.
+
+End the section by thanking the authors that contributed to the release. Get a
+sorted list of contributors in descending order of activity:
+
+```
+git shortlog -sn x.y.z..HEAD | tac | awk '{print $2 " " $3}'
+```
+
+where `x.y.z` is the tag of the previous release. You may need to edit the list
+slightly since not all authors has properly configured their names in git.
+
+*Commit the changes to master.*
+
+
+1.4 Update `CITATION`
+-------------------------------------------------------------------------------
+
+If needed, update the year in `CITATION`.
+
+*Commit the changes to master.*
+
+
+1.5 Create maintenance branch
+-------------------------------------------------------------------------------
+
+*Skip this step if you are preparing a patch release or RC2 and later.*
+
+Create the branch:
+
+```
+git branch x.y
+```
+
+where `x` is the major version number and `y` is the minor version number. Bug
+fixes found in the release candidates should be pushed to master and
+subsequently cherry-picked to the maintenance branch.
+
+*Push branch upstream.*
+
+
+1.6 Prepare and upload archives
+-------------------------------------------------------------------------------
+
+Run autoconf and configure the build:
+
+```
+./autogen.sh
+mkdir build
+cd build
+../configure
+make dist
+```
+
+Rename the archives and generate MD5 sum files:
+
+```
+mv proj-x.y.z.tar.gz proj-x.y.zRCn.tar.gz
+mv proj-x.y.z.zip proj-x.y.zRCn.zip
+md5 proj-x.y.zRCn.tar.gz > proj-x.y.zRCn.tar.gz.md5
+md5 proj-x.y.zRCn.zip > proj-x.y.zRCn.zip.md5
+```
+
+Upload to the OSGeo download server:
+
+```
+scp proj-x.y.zRCn.* warmerdam@download.osgeo.org:/osgeo/download/proj
+```
+
+Adjust version numbers and usernames accordingly.
+
+
+1.7 Announce the release candidate
+-------------------------------------------------------------------------------
+
+Announce the release candidate on the PROJ mailing list.
+
+
+1.8 Promotion to final release
+-------------------------------------------------------------------------------
+
+When you are confident that the latest release candidate is ready for promotion
+to final release a motion for promotion should be issued on the PROJ mailing
+list. Allow for a minimum of 48 hours for voting time.
+
+
+2 Final release
+===============================================================================
+
+Barring a successfull promotion process we can proceed with the final release.
+
+2.1 Prepare and upload archives
+-------------------------------------------------------------------------------
+
+See step 1.6 above. Do not rename the archives with RC postfixes.
```

Comment:
this assumes that no commit has made its way between the last RC and the time we promote it. For GDAL, I usually log in the download server and remove the RCx prefix from the filenames. That way the last RC and the final file are guaranteed to be binary identical

Reply:
That could work too. I've been doing it this way so far because it is quite easy to do. Also, I'd like to setup autotools so that the RCx postfix is added automatically when running make dist. The folder inside the archive would also have the postfix then and the renaming trick would not be so simple anymore. But yes, you have a point, there's a slight change that something was changed by mistake.